src/Services/EncounterService.php

```diff
@@ -725,4 +725,17 @@ public function getEncountersByDateRange($startDate, $endDate)
         }
         return [];
     }
+
+    /**
+     * Returns the default POS code that is set in the facility table.
+     *
+     * @param $facility_id
+     * @return mixed
+     */
+    public function fetchPosCode($facility_id)
+    {
```

Comment:
how about `getPosCode($facility_id)` for consistency :) ?
![soccer](https://github.com/openemr/openemr/assets/5618372/487c2239-a7f2-4667-8808-6f88edf6e2c1)

Reply:
No problem

Reply:
![southpark_cheering](https://github.com/openemr/openemr/assets/5618372/dcbd081f-06e2-4766-a7e5-e4351cb274b7)